src/frontend/src/components/FilterPanel/components/GenomeRecoveryFilter/index.tsx

```diff
@@ -0,0 +1,92 @@
+import Popper from "@material-ui/core/Popper";
+import { Chip, MenuSelect } from "czifui";
+import React, { useEffect, useState } from "react";
+import { DefaultMenuSelectOption } from "../../index";
+import { StyledInputDropdown } from "../../style";
+
+interface Props {
+  updateGenomeRecoveryFilter: (selected?: string) => void;
+}
+
+const GenomeRecoveryFilter = ({
+  updateGenomeRecoveryFilter,
+}: Props): JSX.Element => {
+  const [anchorEl, setAnchorEl] = useState<HTMLElement | null>(null);
+  const [value, setValue] = useState<DefaultMenuSelectOption | null>();
```

Comment:
do you also want to initialize this to null?

Reply:
ooh, ya